packages/chrome/src/styled/sheet/StyledSheetFooter.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_footer';
+
+export interface IStyledSheetFooterProps {
+  /** Sets the SheetFooter padding to half the standard and centers the elements  */
+  isCompact?: boolean;
+}
+
+export const StyledSheetFooter = styled.footer.attrs({
```

Comment:
We're seeing unexpected `isCompact` font-size styling (a 16px Anchor link in Storybook) within this component. However, it could be that the whole `Footer` or maybe the entire `Sheet` should be set to a font size of `theme.fontSizes.md` (?) and the inheritance will cascade as-needed to this child. There are advantages that need to be considered for where to set the font size.

Reply:
Updated, added the font-size to the `StyledSheet`.